packages/breadcrumb/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@undercoatcss/breadcrumbs",
```

Comment:
Decide, breadcrumb**S** or breadcrumb

Reply:
Fixed in 2c471a0b8a946dedd9dc764e0b407586e592ccbf.